lms/templates/problem.html

```diff
@@ -85,9 +85,10 @@ <h3 class="hd hd-3 problem-header" id="${ short_id }-problem-title" aria-describ
             </form>
         % endif
       % endif
-      <div class="submission-feedback" id="submission_feedback_${short_id}">
-        % if attempts_allowed:
-          ${ungettext("You have used {num_used} of {num_total} attempt", "You have used {num_used} of {num_total} attempts", attempts_allowed).format(num_used=attempts_used, num_total=attempts_allowed)}
+      <div class="submission-feedback ${'cta-enabled' if submit_disabled_cta else ''}" id="submission_feedback_${short_id}">
+        ## When attempts are not 0, the CTA above will contain a message about the number of used attempts
+        % if attempts_allowed and (not submit_disabled_cta or attempts_used == 0):
```

Comment:
So basically this is saying that if the CTA is not active, feel free to show it. If it is active, only show this message if there are zero attempts.